### PR TITLE
Update to Documentations

### PR DESCRIPTION
**Pull request by tenorz007**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -17,7 +17,8 @@ Changelog
 - Improved Documentation
 - Minor bug fixes
 
-0.0.6
+1.0.0
```
Comment (prodicus) [inline CHANGELOG.rst]: I just noticed I have bumped it to `1.0.2`. could you make a PR for it?

File: docs/changelog.rst [hunk 2]
```diff
@@ -20,9 +20,10 @@ Changelog
 - Improved Documentation
 - Minor bug fixes
 
-0.0.6
+1.0.0
```
Comment (prodicus) [inline docs/changelog.rst]: as above

**Comment by tenorz007** [comment]
That wasn't a mistake, I know the version is 1.0.2. However I was putting changelog for the changes I had made when you bumped it up to 1.0.0. As I haven't checked to see what changes you made for the 2 additional patches.

**Comment by prodicus** [comment]
@tenorz007 Nothing on the API side as you can see here https://github.com/prodicus/vocabulary/compare/1.0.0...master

**Comment by tenorz007** [comment]
@prodicus So what should be in the changelog then....

**Comment by prodicus** [comment]
@tenorz007 got nothing specific in mind as there wasn't anything specific to the API.

**Comment by tenorz007** [comment]
@prodicus That means there's no notable changes to be added to the change
log then.....

**Comment by prodicus** [comment]
@tenorz007 yup

**Comment by tenorz007** [comment]
@prodicus By the way, I don't think you've been creating releases, just
tags.

**Comment by prodicus** [comment]
Yup. Here is my current workflow.

make all the new changes and commit them to git and then create a tag and
push it to github. Is that the wrong way? Correct me if I am wrong here

Regards,
Tasdik

[image: http://]*tasdikrahman.me*
<http://tasdikrahman.me/>

[image: https://in.linkedin.com/in/tasdikrahman]
<https://in.linkedin.com/in/tasdikrahman>   <http://github.com/prodicus>
<http://www.quora.com/Tasdik-Rahman>
<https://plus.google.com/+TasdikRahman>   <https://twitter.com/tasdikrahman>

**Pull request merged**